.github/workflows/generate-base-image.yml

```diff
@@ -32,9 +36,10 @@ jobs:
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
-      
+
       - name: Generate new tag
         run: |
+          username=${{ secrets.BAL_DOCKER_HUB_USERNAME }}
```

Comment:
This is not always the same, repository and the username can be different. I think its oky to hardcode ballerina here.

Reply:
Done